core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/mercury/v03/request.go

```diff
@@ -154,26 +157,31 @@ func (c *client) multiFeedsRequest(ctx context.Context, ch chan<- mercury.Mercur
 			case http.StatusUnauthorized:
 				retryable = false
 				state = encoding.UpkeepNotAuthorized
+				errCode = encoding.HttpToErrCode(resp.StatusCode)
```

Comment:
seems we can save some lines by moving this assignment outside of switch-case

Reply:
We want the error code just in specific scenarios, being explicit here seems more safe.